tests/foreman/cli/test_host.py

```diff
@@ -1900,304 +1894,6 @@ def test_positive_register(
-        pytest.fail('ContentHostError raised unexpectedly!')
-
-
-@pytest.mark.rhel_ver_match('7')
-@pytest.mark.cli_host_subscription
-@pytest.mark.tier3
-def test_positive_attach_with_lce(
-    module_org,
-    module_ak_with_cv,
-    module_rhst_repo,
-    default_subscription,
-    rhel_contenthost,
-    target_sat,
-):
-    """Attempt to attach a subscription to host, registered by lce
-
-    :id: a362b959-9dde-4d1b-ae62-136c6ef943ba
-
-    :BZ: 1199515
-
-    :customerscenario: true
-
-    :expectedresults: host successfully subscribed, subscription
-        repository enabled, and repository package installed
-
-    :parametrized: yes
-    """
-    res = rhel_contenthost.register(module_org, None, module_ak_with_cv.name, target_sat)
-    assert res.status == 0, f'Failed to register host: {res.stderr}'
-    assert rhel_contenthost.subscribed
-    host = target_sat.cli.Host.info({'name': rhel_contenthost.hostname})
-    target_sat.cli.Host.subscription_attach(
-        {
-            'host-id': host['id'],
-            'subscription-id': default_subscription.id,
-            'quantity': 2,
-        }
-    )
-    rhel_contenthost.enable_repo(module_rhst_repo)
-    # ensure that katello-host-tools can be installed
-    try:
-        rhel_contenthost.install_katello_host_tools()
-    except ContentHostError:
-        pytest.fail('ContentHostError raised unexpectedly!')
-
-
-@pytest.mark.rhel_ver_match('7')
-@pytest.mark.cli_host_subscription
-@pytest.mark.tier3
-def test_negative_without_attach(
-    module_org, module_promoted_cv, module_lce, rhel_contenthost, target_sat
-):
-    """Register content host from satellite, register client to uuid
-    of that content host, as there was no attach on the client,
-    Test if the list of the repository subscriptions is empty
-
-    :id: 54a2c95f-be08-4353-a96c-4bc4d96ad03d
-
-    :expectedresults: repository list is empty
-
-    :parametrized: yes
-    """
-    target_sat.cli.Host.subscription_register(
-        {
-            'organization-id': module_org.id,
-            'content-view-id': module_promoted_cv.id,
-            'lifecycle-environment-id': module_lce.id,
-            'name': rhel_contenthost.hostname,
-        }
-    )
-    host = target_sat.cli.Host.info({'name': rhel_contenthost.hostname})
-
-    rhel_contenthost.register_contenthost(
-        module_org.name,
-        lce=None,  # required, to jump into right branch in register_contenthost method
-        consumerid=host['subscription-information']['uuid'],
-        force=False,
-    )
-    client_status = rhel_contenthost.subscription_manager_status()
-    assert SM_OVERALL_STATUS['current'] in client_status.stdout
-    repo_list = rhel_contenthost.subscription_manager_list_repos()
-    assert "This system has no repositories available through subscriptions." in repo_list.stdout
-
-
-@pytest.mark.rhel_ver_match('7')
-@pytest.mark.cli_host_subscription
-@pytest.mark.tier3
-def test_negative_without_attach_with_lce(
```

Comment:
This still seems like a valid test case to me, although I think it should be renamed to a positive test for 6.16+. Can you please explain your decision to remove it?

Reply:
makes sense, I brought it back with updated name